hazelcast/src/main/java/com/hazelcast/transaction/impl/xa/XATransactionImpl.java

```diff
@@ -78,8 +78,8 @@
     private final SerializableXID xid;
     private final String txOwnerUuid;
 
-    private final List<TransactionRecord> txLogs = new LinkedList<TransactionRecord>();
-    private final Map<Object, TransactionRecord> txLogMap = new HashMap<Object, TransactionRecord>();
+    private final List<TransactionRecord> records = new LinkedList<TransactionRecord>();
```

Comment:
It is a common term and I'm changing the code so it reflects this naming style. Originally an entry in this log (e.g. a map.put) was called a TransactionLog.. but it isn't a lot.. it is a record in the log.